kubecost/templates/_helpers.tpl

```diff
@@ -63,10 +63,12 @@ RBAC exclusivity check: make sure either simple RBAC or RBAC Teams is configured
 {{- end -}}
 
 {{/*
-Print a warning if PV is enabled AND EKS is detected AND the EBS-CSI driver is not installed. This can be disabled is the installer's RBAC does not allow kube-system access
+Print a warning if PV is enabled AND EKS is detected AND the EBS-CSI driver is not installed.
+Skip the check if CI/CD is enabled and skipSanityChecks is set. Argo CD, for
+example, does not support templating a chart which uses the lookup function.
 */}}
 {{- define "kubecost.eksStorage.check" }}
-{{ if (((.Values.kubecost).eksStorage).check).enabled }}
+{{- if not (and .Values.global.platforms.cicd.enabled .Values.global.platforms.cicd.skipSanityChecks) }}
```

Comment:
Delete `eksStorage.check.enabled`, because we already have the guard `skipSanityChecks`. This is the existing guard we use in templates before we perform a Helm `lookup()`